gse-afs-ext-base/src/main/java/com/powsybl/gse/afs/ext/base/ModificationScriptEditor.java

```diff
@@ -48,6 +59,18 @@
 
     private static final ResourceBundle RESOURCE_BUNDLE = ResourceBundle.getBundle("lang.ModificationScript");
 
+    private static final int VALIDATION_INFO_TIMEOUT = 3000;
+
+    private static final ServiceLoaderCache<AbstractCodeEditorFactoryService> CODE_EDITOR_FACTORIES = new ServiceLoaderCache<>(AbstractCodeEditorFactoryService.class);
+
+    private static final List<String> STANDARD_SUGGESTIONS = ImmutableList.of("as", "assert", "boolean", "break", "breaker", "byte",
```

Comment:
As ModificationScriptEditor  could potentially be used with any language, this should probably not be there?

Reply:
I agree.
It used to be part of the default GroovyEditor also, so I didn't bother refactoring that much.